### PR TITLE
[Compute] Added back ProximityPlaceGroup APIs which are missing in 2019-03-01

### PR DESCRIPTION
**Pull request by hyonholee**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5859 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5859/commits/d378bfb417ded9e1fec68c17d5b3c1e0d1ba92a8

**Comment by aspand** [comment]
Looks good

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -337,6 +337,270 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/proximityPlacementGroups/{proximityPlacementGroupName}": {
+      "put": {
+        "tags": [
+          "ProximityPlacementGroups"
+        ],
+        "operationId": "ProximityPlacementGroups_CreateOrUpdate",
```
Comment (veronicagg) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: should this be marked as a long-running operation  "x-ms-long-running-operation": true ?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 2]
```diff
@@ -4477,6 +4741,10 @@
           },
           "description": "A list of references to all virtual machines in the availability set."
         },
+        "proximityPlacementGroup": {
```
Comment (veronicagg) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: Is this property in production already and just missing in the swagger?
Based on ARM guidelines adding a property to a response is a [breaking change](https://sharepoint/sites/AzureUX/Sparta/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B87E6845E-A308-44E8-ACDF-2DB17054BC77%7D&file=RestAPI_BreakingChangeGuide.docx&action=default).

**Review by veronicagg — CHANGES_REQUESTED** [review]
Were these changes deployed with the API and just missing in the swagger?
Have you had ARM team review of these APIs?

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 2 above)
Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: this is already in production and was added in 2018-10-01 API version, but when 2019-03-01 Swagger spec is added, this was missing

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json (hunk 1 above)
Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: @aspand

Comment (hyonholee) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: I just copied from what we have in 2018-10-01 version.

Comment (aspand) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: No ProximityPlacementGroups operation is done in sync path aleady discuused this with Anu in https://github.com/Azure/azure-rest-api-specs/pull/5507

**Comment by hyonholee** [comment]
ARM already approved this change in https://github.com/Azure/azure-rest-api-specs/pull/5507.

This PR is just adding PPG APIs which was added to the previous versions, but is missing in the latest version.

**Review by veronicagg — APPROVED** [review]
Thanks for adding context and replying to my questions, I was not aware of the situation and this API was part of 2018-10-01, so should be good to approve without additional ARM review. Feel free to include this type of comments in the PR from the beginning next time, to expedite approval and merging.

**Pull request merged**